app/views/checkout/thankyou.scala.html

```diff
@@ -13,11 +13,18 @@
 
 @main("Confirmation | Digital | Subscriptions | The Guardian", bodyClasses=List("is-wide"), touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
 
-<main class="page-container gs-container gs-container--slim"
-data-tracking-prop17="GuardianDigiPack:Order Complete"
-data-tracking-products="Subscriptions and Membership;GUARDIAN_DIGIPACK;@{subscriptionProduct.frequency.numberOfMonths};@{subscriptionProduct.price};purchase">
->
-
+<script type="text/javascript">
+    guardian.slug="GuardianDigiPack:Order Complete";
```

Comment:
Would prefer to avoid script blocks in the middle of the markup, could we define these extra properties in a controller (or in scala anyway) and then conditionally expose them in the config in `javaScriptFirstSteps`